### PR TITLE
Update leaver statuses

### PR DESCRIPTION
**Pull request by bodrovis**
I've just noticed that if a player has left the game intentionally, `nil` is being returned for his status. This is because there are more abandonded statuses and he actually gets assigned a status of `3` which is absent in the gem. I've also used strings instead of symbols, but if there is any reason to switch back to symbols, I'll update PR.

### CODE REVIEW

**Comment by vinnicc** [comment]
Okay! I would suggest we stick with symbols though. For the additions, how about:

```
3 => :disconnected
4 => :afk
5 => :never_connected # or :failed_to_connect
6 => :timed_out
```

**Comment by bodrovis** [comment]
Done!

File: lib/dota/api/basic_player.rb [hunk 1]
```diff
@@ -4,17 +4,20 @@ class BasicPlayer < Entity
...
+          2 => :broadcaster,
+          4 => :unassigned
       }
 
       STATUS = {
-        0 => :played,
-        1 => :left_safe,
-        2 => :abandoned,
-        4 => :bot
+          0 => :played,
```
Comment (vinnicc) [inline lib/dota/api/basic_player.rb]: Please fix the spacing, also on `TYPES`. Thanks!

**Comment by bodrovis** [comment]
Sorry, continualtional indents...

**Comment by vinnicc** [comment]
No problem, thanks!

**Pull request merged**